props/saber_fett263_buttons.h

```diff
@@ -4563,9 +4613,40 @@ SaberFett263Buttons() : PropBase() {}
         sound_library_.SaySave();
 #endif
         return true;
-		    
+
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_ON):
+#ifndef DISABLE_COLOR_CHANGE
+        if (DoColorZoom()) return true;
+        if (SaberBase::GetColorChangeMode() == SaberBase::COLOR_CHANGE_MODE_SMOOTH) {
+          SaberBase::SetColorChangeMode(SaberBase::COLOR_CHANGE_MODE_ZOOMED);
+          sound_library_.SayZoomingIn();
+          return true;
+        }
+#endif
+        return false;
+
+      case EVENTID(BUTTON_POWER, EVENT_RELEASED, MODE_ON):
+#ifndef DISABLE_COLOR_CHANGE
+        if (EndColorZoom()) return true;
+        if (SaberBase::GetColorChangeMode() != SaberBase::COLOR_CHANGE_MODE_NONE) {
+          ToggleColorChangeMode();
+#ifdef FETT263_EDIT_MODE_MENU
+          if (menu_type_ == MENU_COLOR) {
+            menu_type_ = MENU_TOP;
+            MenuSave();
+	    return true;
```

Comment:
indentation looks wrong (on github at least)

Reply:
fixed (I think) it looks ok on my end